build/lpc55pins/src/lib.rs

```diff
@@ -98,6 +99,53 @@ pub enum Opendrain {
     Opendrain,
 }
 
+#[derive(Copy, Clone, Debug, Deserialize, PartialEq)]
+#[serde(rename_all = "lowercase")]
+#[repr(u32)]
+pub enum PintSlot {
+    None = 8,
```

Comment:
Having this `PintSlot::None` representation feels very strange. Can we make the API take an `Option<PintSlot>` instead so we don't have to worry about this case?